jarviscli/CmdInterpreter.py

```diff
@@ -174,17 +180,6 @@ def __init__(self, first_reaction_text, prompt, directories=[], first_reaction=T
         self.scheduler = schedule.Scheduler()
         self.speech = create_voice()
 
-        self.actions = [{"check": ("ram", "weather", "time", "forecast")},
-                        "directions",
-                        "help",
-                        "how_are_you",
-                        "near",
-                        "pinpoint",
-                        "umbrella",
-                        {"update": ("location", "system")},
-                        "weather",
-                        ]
-
         self.fixed_responses = {"what time is it": "clock",
                                 "where am i": "pinpoint",
                                 "how are you": "how_are_you"
```

Comment:
Not related to your PR, but I think this has been removed

Reply:
Ok I found it in `plugin/ask.py` line 63.

But its not working if i dont give extra string to it

```
16:59 $ python jarviscli/
Jarvis' sound is by default disabled.
In order to let Jarvis talk out loud type: enable sound
Type 'help' for a list of available actions.
86 plugins loaded 4 plugins disabled. More information: status

~> Hi, what can I do for you?
how are you
I could not identify your command...
~> What can i do for you?
how are you jarvis
I am fine, How about you?
~> What can i do for you?
how are you 1
I am fine, How about you?
~> What can i do for you?


```